src/motion_input_shaping/shaped_axis.py

```diff
@@ -9,9 +9,11 @@
 
 import time
 import sys
-from zaber_motion import Units
+from zaber_motion import Units, Measurement
 from zaber_motion.ascii import Connection, Axis
 from zero_vibration_shaper import ZeroVibrationShaper
+from zero_vibration_stream_generator import ZeroVibrationStreamGenerator, ShaperType
+from shaper_config import ShaperConfig, ShaperMode
 
 
 class ShapedAxis(Axis):
```

Comment:
Followup from Martin's discussion this morning. We should change this to use composition instead of inheritance

Reply:
Done